### ISSUE TITLE
Single onward journey - VHT end of component

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#689]
**Is your feature request related to a problem? Please describe.**
After the components content there should be a `div` element with visually hidden text stating "End of ${title}." EG: "End of More on Archaeology."

**Describe the solution you'd like**
Put an instance of the Visually Hidden Text component as the last thing in the component.
- A `div` element should contain the visually hidden text (Note when testing the implementation a div rather than a span makes Jaws/IE read out the correct content when using the skip link)
- Note the full stop makes the screen reader (VoiceOver iPhone) acknowledge a pause when reading the content out in one go
- The element will need to have an id that maps to the skip link `id="end-of-more-on-archaeology'`, see https://github.com/BBC-News/simorgh/issues/685
- The element will need a tabindex value of -1 `tabindex="-1"` in order for the skip link to work correctly with assistive technology

**Screenshot**
![image](https://user-images.githubusercontent.com/7791726/45832944-3e218f00-bcfb-11e8-8105-5c9fa127758c.png)
